crates/ty/src/main.rs

```diff
@@ -306,6 +306,10 @@ impl MainLoop {
                                 if diagnostics_count > 1 { "s" } else { "" }
                             )?;
 
+                            if max_severity.is_fatal() {
+                                tracing::warn!("A panic occurred while checking some files. Not all project files were analyzed. See the diagnostics list above for details.");
```

Comment:
Could we mention that the panic will be the diagnostic with the `[fatal]` severity? It might make it easier for users to figure out which one in the diagnostic list was the panic

Reply:
I can try

Reply:
I would just say "A fatal error occurred" instead of "A panic occurred" -- I think that makes it clear enough that you're looking for the error with `[fatal]` severity. And "panic" is kind of Rust-specific terminology.